whylog/teacher/__init__.py

```diff
@@ -159,6 +140,20 @@ def set_causes_relation(self, relation):
         """
         pass
 
+    def make_group(self, line_id, span):
```

Comment:
I suggest to not add dead code. This, as well as the next function, should be added when we have some idea on to what to do with it next. If we commit this, we can also commit a lot of other stuff and eventually end up with half of code not being used for anything.